src/components/CurrentUser/CurrentUser.tsx

```diff
@@ -1,12 +1,61 @@
 import React from 'react';
 import './CurrentUser.scss';
+import { getUser } from '../../api';
 
-export const CurrentUser: React.FC = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+type Props = {
+  selectedUserId: number;
+  clearUser: (userId: number) => void;
+};
 
-    <h3 className="CurrentUser__name">Ervin Howell</h3>
-    <p className="CurrentUser__email">Shanna@melissa.tv</p>
-    <p className="CurrentUser__phone">010-692-6593 x09125</p>
-  </div>
-);
+type State = {
+  user: User;
+};
+
+export class CurrentUser extends React.Component<Props, State> {
+  state: State = {
+    user: {
+      id: 0,
+      name: '',
+      email: '',
+      phone: '',
+    },
+  };
+
+  async componentDidMount() {
+    const user = await getUser(this.props.selectedUserId);
+
+    this.setState({ user });
```

Comment:
you repeat this code here and in componentDidUpdate, create a separate method and call it where needed (also it will help with a linter issue)